integration-fe/src/views/CreateView.jsx

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+
+class CreateView extends Component {
+
+  componentDidMount() {
```

Comment:
For sure. I've created this method to avoid warnings from es lint. I will change this.